app/models/patron_request.rb

```diff
@@ -195,6 +200,8 @@ def earliest_delivery_estimate(scan: false)
     if any_items_avaliable?
       paging_info = PagingSchedule.for(self, scan:).earliest_delivery_estimate
       { 'date' => Date.parse(paging_info.to_s), 'display_date' => paging_info.to_s }
+    elsif title_only?
+      { 'date' => Time.zone.today + 1.year, 'display_date' => 'No date/time estimate' }
```

Comment:
I don't think this actually necessary. I think you can just add a year to line after.

Reply:
Based on discussion, I made a partial for the not needed after date section, since it's also used in request preferences.  I moved this part of the method out as no longer needed.